### PR TITLE
Adding new CA trusted certificates

### PR DESCRIPTION
**Pull request by CIPop**
Adding new CA certificates and updating the documentation to create a store when testing in Windows.

### CODE REVIEW

**Comment by momuno** [comment]
Readme looks good.  Thanks for your note about how to convert the crt to the pem.

File: sdk/samples/iot/README.md [hunk 1]
```diff
@@ -329,14 +329,29 @@ Set the following environment variables for all samples:
       $env:VCPKG_ROOT='<FULL PATH to vcpkg>'
       ```
 
-  2. Set the trust pem filepath. **Only for Windows or if required by OS.**
+  2. Set the trust pem filepath. **Only when testing on Windows.**
 
-      Download [BaltimoreCyberTrustRoot.crt.pem](https://cacerts.digicert.com/BaltimoreCyberTrustRoot.crt.pem) to `<FULL PATH TO azure-sdk-for-c>\sdk\samples\iot\`. Confirm the downloaded certificate uses the correct file name and file extension.
+      _Important:_ We recommend using a managed trusted store for production purposes. Paho/OpenSSL on Windows is meant for testing purposes only.
```
Comment (ericwol-msft) [inline sdk/samples/iot/README.md:334]:
```suggestion
      _Important:_ We recommend using a managed trusted store for production deployments. Paho/OpenSSL on Windows is meant for testing purposes only.
```

File: sdk/samples/iot/README.md [hunk 2]
```diff
@@ -329,14 +329,29 @@ Set the following environment variables for all samples:
...
+            
+      Create a PEM certificate store by concatenating the following files:
```
Comment (ericwol-msft) [inline sdk/samples/iot/README.md:336]:
```suggestion
      Create a PEM certificate file based store by concatenating the following files:
```

**Review by ericwol-msft — APPROVED** [review]

**Pull request merged**